lib/user/user-session.js

```diff
@@ -0,0 +1,59 @@
+var express = require('express')
+  , config = require('./config-user')
+  , MbaasServiceProxy = require('./mbaas-service-proxy');
+
+function initRouter(msProxy) {
+  var router = express.Router();
+
+  router.route('/auth').post(function(req, res, next) {
+    var params = req && req.body && req.body.params;
+    msProxy.auth(params).then(function(data) {
+      res.json(data);
+    }, function(error) {
+      next(error);
+    });
+  });
+
+  router.route('/verifysession').post(function(req, res, next) {
+    var fhParams = req.fh_params || {__fh: {}};
+    var sessionToken = fhParams.__fh.sessionToken || fhParams.__fh.sessiontoken;
+    msProxy.verifysession(sessionToken).then(function(data) {
+      res.json(data);
+    }, function(error) {
+      next(error);
+    });
+  });
+
+  router.route('/revokesession').post(function(req, res, next) {
+    var fhParams = req.fh_params || {__fh: {}};
+    var sessionToken = fhParams.__fh.sessionToken || fhParams.__fh.sessiontoken;
+    msProxy.revokesession(sessionToken).then(function(data) {
+      res.json(data);
+    }, function(error) {
+      next(error);
+    });
+  });
+
+  return router;
+}
+
+module.exports = function(mediator, app, guid) {
```

Comment:
`guid` should really be called `serviceGuid` as the requests use the $fh.service API.